packages/component/src/Utils/activityAltText.ts

```diff
@@ -0,0 +1,40 @@
+import isSSML from './isSSML';
+
+/** Retrieves all text nodes from a given DOM tree as flattened array. */
+function allTextContents(element: Node): string[] {
+  const nodes: Node[] = [].slice.call(element.childNodes);
+  const results: string[] = [];
+
+  while (nodes.length) {
+    const { childNodes, nodeType, textContent } = nodes.shift();
+
+    if (nodeType === Node.TEXT_NODE) {
+      const trimmedTextContent = textContent.trim();
+
+      trimmedTextContent && results.push(trimmedTextContent);
+    } else {
+      nodes.unshift(...[].slice.call(childNodes));
+    }
+  }
+
+  return results;
+}
+
+/** Returns the alt text for an activity. */
```

Comment:
In the future, you may want to use JS Docs format:
```
/*
 * <description>
 @params
*/
```

Reply:
I think `<description>` is not JSDoc thing. But will add `@params`.